src/Formatter/PhpCsFixerFormatter.php

```diff
@@ -36,16 +35,6 @@ public function format(Process $process): string
         return $this->formatJsonResponse($json);
     }
 
-    public function formatSuggestion(Process $process): string
```

Comment:
Agree we can get rid of this method. However, we should still suggest the fix command. (See further in CR)